plugin/core/main.py

```diff
@@ -16,12 +18,18 @@
     pass
 
 
+def ensure_server_panel(window: sublime.Window) -> 'Optional[sublime.View]':
+    return ensure_panel(window, "server", "", "", "Packages/LSP/Syntaxes/ServerLog.sublime-syntax")
```

Comment:
The "server" name for the panel is a bit ambiguous. Not clear that it's LSP's logging.

I mean when looking here:
<img width="187" alt="Screenshot 2019-12-09 at 14 43 22" src="https://user-images.githubusercontent.com/153197/70440358-4b5aa300-1a92-11ea-881d-9e76b48bb737.png">

Reply:
Perhaps: "Language Servers" ?